src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -596,7 +596,7 @@
     <Compile Include="$(CommonPath)System\Threading\Tasks\TaskToApm.cs"
              Link="Common\System\Threading\Tasks\TaskToApm.cs" />
   </ItemGroup>
-  <ItemGroup Condition="('$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true') and '$(TargetsOSX)' != 'true' and '$(TargetsiOS)' != 'true' and '$(TargetstvOS)' != 'true' and '$(TargetsBrowser)' != 'true'">
+  <ItemGroup Condition="'$(TargetsUnix)' == 'true' and '$(TargetsOSX)' != 'true' and '$(TargetsiOS)' != 'true' and '$(TargetstvOS)' != 'true' and '$(TargetsBrowser)' != 'true'">
```

Comment:
```suggestion
  <ItemGroup Condition="'$(TargetsUnix)' == 'true' and '$(TargetsOSX)' != 'true' and '$(TargetsMobile)' != 'true'">
```

Reply:
This will lead to fail on android, as we are using the linux implemenation for android

Reply:
You're correct. I missed the fact that we were not excluding for Android here. We can change to use TargetsMobile != true when we don't want to include on Android, iOS, tvOS and Browser.